docker/openlane/Dockerfile

```diff
@@ -69,4 +69,6 @@ COPY ./.tclshrc /root
 
 WORKDIR $OPENLANE_ROOT
 
+RUN dbus-uuidgen --ensure
```

Comment:
Just curious what this is for?

Reply:
I think Qt needs it?

Reply:
Its not part of the OR build afaik.

Reply:
Yeah iunno, without this command it would tell me there's a missing uuid.